### PR TITLE
VZ-9965: Add ability uptake to new OCNE versions without upgrading Verrazzano

### PR DESCRIPTION
**Pull request by mgianatagh**
Provide a summary of the change and which issue (i.e. ticket) is fixed.
If there are any dependencies, for example on a PR in another repository, list those.

### CODE REVIEW

File: platform-operator/controllers/verrazzano/component/clusterapi/clusterapi_component.go [hunk 1]
```diff
@@ -154,8 +172,8 @@ func (c clusterAPIComponent) IsOperatorInstallSupported() bool {
 
 // IsInstalled checks to see if ClusterAPI is installed
 func (c clusterAPIComponent) IsInstalled(ctx spi.ComponentContext) (bool, error) {
-	daemonSet := &appsv1.Deployment{}
-	err := ctx.Client().Get(context.TODO(), types.NamespacedName{Namespace: ComponentNamespace, Name: capiCMDeployment}, daemonSet)
+	deployment := &appsv1.Deployment{}
```
Comment (jmaron99) [inline platform-operator/controllers/verrazzano/component/clusterapi/clusterapi_component.go:175]: curious what supured the change from daemon set to deployment?

Comment (michael-cico) [reply platform-operator/controllers/verrazzano/component/clusterapi/clusterapi_component.go:175]: looks like it was always a deployment, but the variable was named `daemonSet`.  Looks like it just fixed the var name

File: platform-operator/controllers/verrazzano/component/clusterapi/clusterapi_overrides.go [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+	}
+	// Return default value
+	return formatProviderURL(false, provider.Image.Repository, provider.Name, provider.Image.BomVersion, provider.MetaddataFile)
+}
+
+// formatProviderURL - return the provider URL using the following format
+// https://github.com/{owner}/{Repository}/releases/{version-tag}/{componentsClient.yaml}
+func formatProviderURL(remote bool, owner string, repo string, version string, metadataFile string) string {
+	if remote {
+		return fmt.Sprintf("https://github.com/%s/%s/releases/%s/%s", owner, repo, version, metadataFile)
```
Comment (michael-cico) [inline platform-operator/controllers/verrazzano/component/clusterapi/clusterapi_overrides.go:188]: Will assuming `github.com` work for the air-gapped case, or is this something that's not expected to work in an air-gapped env?

Comment (michael-cico) [reply platform-operator/controllers/verrazzano/component/clusterapi/clusterapi_overrides.go:188]: OK, I see it's guarded by a `remote` flag?

File: platform-operator/controllers/verrazzano/component/clusterapi/clusterapi_component.go (hunk 1 above)
Comment (mgianatagh) [reply platform-operator/controllers/verrazzano/component/clusterapi/clusterapi_component.go:175]: Yes, just fixing the variable name.

File: platform-operator/controllers/verrazzano/component/clusterapi/clusterapi_overrides.go (hunk 2 above)
Comment (mgianatagh) [reply platform-operator/controllers/verrazzano/component/clusterapi/clusterapi_overrides.go:188]: Yes, you only get to this function if expecting to pull from the internet, or using the default path (which is baked into the VPO image).  Remote is true when expecting to pull from GitHub.

**Review by michael-cico — APPROVED** [review]

**Pull request merged**